cmd2/cmd2.py

```diff
@@ -466,11 +462,33 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, persistent
         # If this string is non-empty, then this warning message will print if a broken pipe error occurs while printing
         self.broken_pipe_warning = ''
 
+        # Commands that will run at the beginning of the command loop
+        self._startup_commands = []
+
         # If a startup script is provided, then add it in the queue to load
         if startup_script is not None:
             startup_script = os.path.abspath(os.path.expanduser(startup_script))
             if os.path.exists(startup_script) and os.path.getsize(startup_script) > 0:
-                self.cmdqueue.append("load '{}'".format(startup_script))
+                self._startup_commands.append("load '{}'".format(startup_script))
+
+        # Transcript files to run instead of interactive command loop
+        self._transcript_files = None
```

Comment:
This should be set to `transcript_files` and the `elif` below should be deleted.  The intention is that transcript files can be provided in the call to `__init__` regardless of whether or not `allow_cli_args` is true, but if `allow_cli_args` is true **and** the `-t / --test` flag was provided, then that would override whatever was passed to `__init__`